fcrepo-search-api/src/main/java/org/fcrepo/search/api/PaginationInfo.java

```diff
@@ -21,24 +21,44 @@
 
 /**
  * A data structure representing the pagination information associated with a {@link org.fcrepo.search.api.SearchResult}
+ *
  * @author dbernstein
  */
 public class PaginationInfo {
     @JsonProperty
-    int totalResultCount = 0;
-
+    private int offset = -1;
     @JsonProperty
-    int startIndex = -1;
+    private int maxResults = -1;
 
-    @JsonProperty
-    int endIndex = -1;
+    /**
+     * Default constructor
+     */
+    public PaginationInfo() {}
```

Comment:
An empty default public constructor is implicit. This can be removed.

Reply:
It's not implicit because there's another constructor defined. That said, I can't find the no-arg constructor being used anywhere.

Reply:
It seems to be required for the json serialization to work.